mathesar/api/serializers/constraints.py

```diff
@@ -121,6 +136,13 @@ class Meta:
 
     def create(self, validated_data):
         serializer = self.get_serializer_class(self.get_mapping_field(validated_data))
+        if serializer is None:
+            message = 'Unknown type passed.'
+            raise base_api_exceptions.NotFoundAPIException(
```

Comment:
Replace it with the correct error class, check `mathesar.api.exceptions.validation_exceptions.exceptions.UnknownDatabaseTypeIdentifier` for reference